### ISSUE TITLE
Service port is not reflect on ingress

### ISSUE THREAD

**Issue by carlosmathiasen987** [YOURLS/charts#48]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Self troubleshooting

- [X] I've read the [documentation](https://yourls.org/) and made sure to follow it.
- [X] I've read the [troubleshooting first steps](https://github.com/YOURLS/YOURLS/wiki/Troubleshooting-first-steps) guide.

### Version

1.8.1

### Description

If I choose a different than 80 port for mi service, the ingress not recognize the service, because in the "ingress template"  the servicePort is hardcoded to 80

**values.yaml**
```
service:
  type: ClusterIP
  port: 8080
```

**templates/ingress.yaml**
```
  - host: {{ .name }}
    http:
      paths:
        - path: {{ default "/" .path }}
          backend:
            serviceName: "{{ template "yourls.fullname" $ }}"
            servicePort: 80
```

### Expectation

When you change the port in the service, the ingress should see that service

**values.yaml**
```
service:
  type: ClusterIP
  port: 8080
```
### Posible solution: 
Adding the value from values.yaml is exists.

**templates/ingress.yaml**
```
  - host: {{ .name }}
    http:
      paths:
        - path: {{ default "/" .path }}
          backend:
            serviceName: "{{ template "yourls.fullname" $ }}"
            servicePort: {{if .Values.service.port}}{{.Values.service.port}}{{else}}80{{end}}
```

**Comment by LeoColomb** [comment]
Oh wow, really good catch @carlosmathiasen987, thanks for that!
Would you like to open a pull request?

**Comment by LeoColomb** [comment]
Closed via c389b553faaecef49dae5277d3ec61c274891301
